examples/plugin/src/test/groovy/acceptance/ResponseSpec.groovy

```diff
@@ -41,7 +43,7 @@ class ResponseSpec extends Specification implements WireMockPerTest, PetApiDsl,
             fetchingPet()
 
         then:
-            thrown(ApiException)
+            thrown(ProcessingException)
```

Comment:
I suppose this is a bit of a regression on openapi's part because they aren't catching the exception and they're exposing the underlying jersey exception